test/unit/org/apache/cassandra/index/sai/cql/VectorInvalidQueryTest.java

```diff
@@ -20,6 +20,7 @@
 
 import java.util.Collections;
 
+import com.datastax.driver.core.exceptions.InvalidQueryException;
```

Comment:
Not needed, we check for `InvalidRequestException`

Reply:
Removed.